build/sdk_update.go

```diff
@@ -310,3 +295,67 @@ func fixGitFolder(path string) string {
 func isDotGit(path string) bool {
 	return strings.Contains(path, ".git/")
 }
+
+// restartSDKUpdate executes a driver SDK update as an external command.
+// This is needed when an update is triggered by a Go script and changes a major SDK version.
+// The function will then re-run the Go script, so the new SDK version is used for it.
+func restartSDKUpdate(root string) error {
+	return goExec(root, "run", "./update.go")
+}
+
+func updateToModules(root string) error {
+	m, err := manifest.Load(filepath.Join(root, manifest.Filename))
+	if err != nil {
+		return err
+	}
+	// first, generate Go module files
+	if err := modInit(root, "github.com/bblfsh/"+m.Language+"-driver"); err != nil {
+		return err
+	}
+	if err := modTidy(root); err != nil {
+		return err
+	}
+	if err := modVendor(root); err != nil {
+		return err
+	}
+	// next, replace Go version in the driver's manifest
+	if err = updateGo(root, "1.12"); err != nil {
```

Comment:
Is this necessary? The only place this tag matters is if the Go standard library changes in incompatible ways, and it doesn't prevent earlier (or later) versions from building the module.

Reply:
This is not the change for `go.mod`, it's a change to the driver manifest that affects what version is used to build the driver. So yes, I think updating it to 1.12 matters.